tox.ini

```diff
@@ -1,11 +1,18 @@
 [tox]
 minversion = 2.3.1
-envlist = flake8,pre-commit
+envlist = flake8,pre-commit,tests
 
 [testenv]
```

Comment:
Why wouldn't we just use this testenv?

Reply:
You mean like this? Sorry, I don't normally use tox so not sure what the conventions are..

```
[testenv]
deps = flake8
       pytest
commands =
    pytest tests {posargs}
```

Reply:
Yes, instead of the tests enc, just use this one